src/plugin_discovery/plugin_config/schema.js

```diff
@@ -0,0 +1,25 @@
+import Joi from 'joi';
+import { noop } from 'lodash';
```

Comment:
nit: Seems like a trivial reason to make this module depend on lodash when an empty arrow function would suffice.